BitwardenShared/UI/Autofill/Utilities/TestHelpers/Fido2UserVerifiableCipherView+Fixtures.swift

```diff
@@ -0,0 +1,13 @@
+// swiftlint:disable:this file_name
+
+import BitwardenSdk
+
+@testable import BitwardenShared
+
+struct MockFido2UserVerifiableCipherView: Fido2UserVerifiableCipherView {
+    var reprompt: BitwardenSdk.CipherRepromptType
+
+    init(reprompt: BitwardenSdk.CipherRepromptType = .none) {
+        self.reprompt = reprompt
+    }
```

Comment:
🎨 Since this is a struct, you can assign a default value and get the same initializer for free.
```suggestion
struct MockFido2UserVerifiableCipherView: Fido2UserVerifiableCipherView {
    var reprompt: BitwardenSdk.CipherRepromptType = .none
```